action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -291,13 +303,63 @@ def _parse_transfer(transaction: dict, **kwargs) -> Optional[dict]:
         "tx_index": kwargs.get('tx_index', 'N/A'),
     }
 
+def parse_no_reports_report(all_reports) -> dict[str, dict]:
```

Comment:
General docstring would be good here to show an example of input params and what's the output of this function

Reply:
Still see no docstring and `all_reports`  typing is missing :)